test/integration/testMaxLobSize.js

```diff
@@ -0,0 +1,240 @@
+const assert = require('assert');
+const async = require('async');
+const testUtil = require('./testUtil');
+const crypto = require('crypto');
+const snowflake = require('./../../lib/snowflake');
+const connOption = require('./connectionOptions');
+
+function generateRandomString(sizeInBytes) {
+  const bufferSize = Math.ceil(sizeInBytes / 2);
+  const buffer = crypto.randomBytes(bufferSize);
+  return buffer.toString('hex').slice(0, sizeInBytes);
+}
+
+describe('Max LOB test', function () {
```

Comment:
We need to add here condition like ```if (process.env.RUN_MANUAL_TESTS_ONLY === 'true')``` because this test can't work on all environments nowadays.

Reply:
Added.